parmed/openmm/parameters.py

```diff
@@ -213,6 +224,7 @@ def from_parameterset(cls, params, copy=False, remediate_residues=True):
         new_params._combining_rule = params.combining_rule
         new_params.default_scee = params.default_scee
         new_params.default_scnb = params.default_scnb
+        new_params.unique_atom_types = unique_atom_types
```

Comment:
`unique_atom_types` is only set in this constructor, but is used like it's always present below (unless I'm missing something).  It should probably be set to `False` in `__init__` as well to make sure the attribute is always present.